src/Tribe/Views/V2/Views/Recent_Past_View.php

```diff
@@ -125,25 +155,20 @@ protected function setup_repository_args( Tribe__Context $context = null ) {
 	 */
 	public function add_view_filters() {
 
-		add_filter( 'tribe_template_html:components/messages', [ $this, 'filter_template_done' ] );
-		add_filter( 'tribe_template_html:events/v2/day', [ $this, 'add_view' ] );
-		add_filter( 'tribe_template_html:events/v2/list', [ $this, 'add_view' ] );
-		add_filter( 'tribe_template_html:events/v2/month', [ $this, 'add_view' ] );
-
-		// PRO Views
-		add_filter( 'tribe_template_html:events-pro/v2/map', [ $this, 'add_view' ] );
-		add_filter( 'tribe_template_html:events-pro/v2/photo', [ $this, 'add_view' ] );
-		add_filter( 'tribe_template_html:events-pro/v2/week', [ $this, 'add_view' ] );
+		add_filter( 'tribe_template_html:events/v2/components/messages', [ $this, 'filter_template_done' ] );
+		add_filter( 'tribe_template_html:events/v2/components/ical-link', [ $this, 'add_view' ] );
 	}
 
 	/**
 	 * Connect Whitelist Filter to Tribe Template Done.
 	 *
```

Comment:
Since this function is `public`, it will be documented.
As such, please, add a long description, and maybe review the short description to detail what is doing and why (not the how, though).